### ISSUE TITLE
Torrent creation function not working

### ISSUE THREAD

**Issue by Rakleed** [picotorrent/picotorrent#1050]
When I try to create a torrent, I get the error `no files in torrent`. I tried to create a torrent for both a folder and a separate file. I filled in all the fields in the create torrent dialog, but every time I got the same error. In this case, the error `error: Error when setting piece hashes: libtorrent:16` appears in the log each time. Please fix this problem.

**Comment by vktr** [comment]
Do you have read access to the files you select? Also, does it crash, and if so do you submit the crash report?

**Comment by Rakleed** [comment]
Yes, I can read, modify and delete these files. No, it doesn't crash. Only what I wrote in the first post happens.

**Comment by vktr** [comment]
Thanks for the clarification!

**Comment by OmnyH** [comment]
If the file is at the root of the drive, there is no longer a problem.

**Comment by vktr** [comment]
Thanks for the additional information!
